File: C#/13-Métodos/Metodo.cs
using System;

class Metodo
{
//Método: acesso tipo_retorno nomeMétodo(parâmetros)
//Vamos criar um método que escreve na tela Hello World
//Nome do método sempre vem antes de ().
//Void indica que método não retorna nenhum dado armazenado.
    public static void HelloWorld()
    {
        Console.WriteLine("Hello World.");
    }

//Aqui se inicia o método principal(Main) onde o programa começa a ser executado
    static void Main(string[] args)
    {
//Vamos chamar o método criado sempre seguido de ()
        HelloWorld();
        Console.ReadLine();
    }
}

File: Java/13-Métodos/Metodo.java
class Metodo {
    // Metodos são blocos de códigos que podem ser chamados através do seu nome.
    // Vamos criar um que escreve ola mundo na tela(sem necessidade de parametros)

    static void ola() { // void não retorna nada(procedimento)
        System.out.println("Ola mundo"); // static método pertence a classe
    }

    // Metodo principal só aceita procedimentos static
    public static void main(String[] args) {
        // Toda vez que metodo for chamado executa seu bloco de comandos
        System.out.println("Chamando metodo");
        ola(); // método deve ser static para ser chamado aqui pois esta classe é static
        System.out.println("Chamando metodo");
        ola();
    }
}